src/outputs.cc

```diff
@@ -1230,13 +1230,16 @@ void SeriesData(struct site source, struct site destination, unsigned char fresn
                     fpt6_zone = 0.0, nm = 0.0, nb = 0.0, ed = 0.0, es = 0.0, r = 0.0, d = 0.0, d1 = 0.0, terrain, azimuth,
                     distance, minterrain = 100000.0, minearth = 100000.0;
     struct site remote;
+    struct site mp;
 
     bool has_clutter = false, has_fresnel = false;
 
     ReadPath(destination, source, out);
     azimuth = Azimuth(destination, source);
     distance = Distance(destination, source);
     refangle = ElevationAngle(destination, source);
+    printf("refangle is %f\n", refangle);
+    refangle = 0.0;
```

Comment:
why zeroize immediately?

Reply:
this was all experimental, I was *trying* to center the peak of the earth curve, but zeroizing at least let me get the transmitter (called the destination here, for some reason) at the top of the curve, most of the time.